src/index.jsx

```diff
@@ -49,27 +55,77 @@ function Form() {
+        if (item) {
+            setApp(item.key);
+        }
+    };
+
+    const changeOpenState = (item) => {
+        if(isDropdownOpen == true){
+            document.getElementById("root").className = "normal";
+            document.getElementById("popoutBody").className = "normal";
+            setDropdownOpen(false);
+        } else {
+            document.getElementById("root").className = "expanded";
+            document.getElementById("popoutBody").className = "expanded";
+            setDropdownOpen(true);
+        }
+        
+    };
+
     return <Box display="flex" flexDirection="column">
+        {/* https://datahub.berkeley.edu */}
         <Heading sx={{ fontSize: 2, mb: 1 }}>JupyterHub URL</Heading>
 
-        <TextInput value={hubUrl} onChange={(ev) => setHubUrl(ev.target.value)} placeholder="https://myjupyterhub.org" aria-label="JupyterHub URL" />
+        <TextInput
+            value={hubUrl}
+            onChange = {
+                (ev) => setHubUrl(ev.target.value)
+            }
+            placeholder="https://myjupyterhub.org"
+            aria-label="JupyterHub URL"
+            sx={{ pt: 0.5, pb: 0.5 }}
+        />
+
         <Text color="danger.fg" sx={{ visibility: isValidHubUrl ? "hidden" : "visible" }}>Enter a valid URL</Text>
 
-        <Heading sx={{ fontSize: 2, mb: 1, mt: 2 }}>Open in</Heading>
-        <select className="form-select mb-1" onChange={(ev) => setApp(ev.target.value)} value={app}>
-            {Object.entries(AVAILABLE_APPS).map(([name, value]) => {
-                return <option key={name} value={name}>{value.title}</option>
-            })};
-        </select>
-
-        <Button disabled={!isValidHubUrl || finishedCopying} sx={{ mt: 2 }} onClick={() => {
-            copyGeneratedUrl(hubUrl, app);
-            // Flash a 'Copied!' message for 3 seconds after copying
-            setFinishedCopying(true);
-            setTimeout(() => setFinishedCopying(false), 3 * 1000)
-        }}>
-            <CopyIcon /> {finishedCopying ? "Copied!" : "Copy nbgitpuller link"}
-        </Button>
+        <Heading sx={{ fontSize: 2, mb: 1, mt: 3 }}>Open in</Heading>
+
+        <DropdownMenu
+            items={menuItems}
+            onChange={handleMenuChange}
+            placeholder={AVAILABLE_APPS[app].title} // Display the value of 'name' or a placeholder
+            open={isDropdownOpen}
+            onOpenChange={changeOpenState}
+        />
+
+        <div style={{ display: 'flex', gap: '8px', justifyContent: 'center' }}>
+
+            <Button disabled={!isValidHubUrl || finishedCopying} sx={{ mt: 2}} onClick={() => {
+                copyGeneratedUrl(hubUrl, app, true);
+            }}>
+                <TabExternalIcon /> Open in tab
+            </Button>
+
+            <Button disabled={!isValidHubUrl || finishedCopying}
+                style={{ flexGrow: 1, backgroundColor: '#52c786', opacity: ((!isValidHubUrl || finishedCopying)?0.6:1),  marginTop: '8px' }}
```

Comment:
This should also be CSS classes, instead of inline styles. Can you also match the same green GitHub uses for its primary action?

<img width="252" alt="image" src="https://github.com/yuvipanda/nbgitpuller-link-generator-webextension/assets/30430/87836448-876d-49da-826d-30dcde1eca12">

Does setting this button's `disabled` state not change opacity or some other visual marker? I suspect that maybe setting `backgroundColor` directly (instead of using something else primer native?) may also be what's causing that?

Reply:
the disabled state does not change color.

Reply:
fixed using CSS